jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/HttpDateTime.java

```diff
@@ -185,7 +185,7 @@ else if (token.length() == 4)
         catch (Throwable x)
         {
             if (LOG.isDebugEnabled())
-                LOG.debug("Ignore: Unable to parse Date/Time", x);
+                LOG.atDebug().setCause(x).log("Ignore: Unable to parse Date/Time");
```

Comment:
Not necessary here, you are in a catch block already.
The Throwable will exist.